src/components/Tabnav.vue

```diff
@@ -9,7 +9,10 @@
 -->
 
 <template>
-  <nav class="tabnav">
+  <nav
+    class="tabnav"
+    :class="[{ [`tabnav--${position}`]: position, 'tabnav--vertical': vertical }]"
```

Comment:
I believe it should work without wrapping the object in an array

Reply:
yes you are right.